src/cli.rs

```diff
@@ -89,6 +89,10 @@ pub enum Command {
         /// Default append port to the base url.
         #[clap(long)]
         no_port_append: bool,
+
+        /// Extra path to watch for changes, relative to the project root.
+        #[clap(long)]
+        extra_watch_path: Vec<String>,
```

Comment:
Should it be `extra_watch_paths` since it's a vec?

Reply:
It's a vec, but the user would multiple paths by repeating the argument.
```bash
zola serve --extra_watch_path="a" --extra_watch_path="b"
```
So I think the singular makes the most sense. There's some precedent for this, e.g.
```bash
curl --header "Content-Type: application/json" --header "Authorization: Bearer token" https://api.example.com
grep --regexp="pattern1" --regexp="pattern2" file.txt
```
Happy to change it if you prefer though.

Reply:
That's fine, I thought it was `--extra_watch_paths=something,something_else` with clap being smart enough to split on the `,`